### DISCUSSION TITLE
Introductions!

### DISCUSSION THREAD

**Discussion by samnang** [web3together/web3together#2] [Announcements]
Hey everyone! This thread is for introducing yourself to the community! Feel free to to post whatever you’d like but be sure to include:

* Your name
* What do you do right now?
* How much experience do you have building in web3?
* What platforms or frameworks are you most familiar with?

Please note [Code of Conduct](https://github.com/web3together/web3together/blob/main/CODE_OF_CONDUCT.md).

**Comment by samnang** [comment]
Let me start by introducing myself:

* Samnang Chhun (You can connect me on [twitter](https://twitter.com/samnangchhun))
* CTO of @somleng and Co-CTO at Kenorei
* Web3 development is pretty new for me. I have looked at it recently as well, but I looked at the blockchain and understood the foundation since a few years ago. I completed a few courses and books around the topic.
* I have been programming for 15+ years now. For the last 10 years, Ruby and Rails are the most platforms and frameworks that I have been working on.

**Reply by metrey** [reply]
Interesting somleng, is it opensource that we can deploy for commercial use ?

**Reply by samnang** [reply]
@metrey for sure, it's in MIT, so you use it in commercial or fork it to modified it. MIT license allowes you to use whatever way you like.

**Comment by BunHouth** [comment]
Let me start by introducing myself:

- Bunhouth (Bun is shorter )
- Coder  😬😬 
- Web3 really fresh but love to work on it.
- I have been programming for 6+ years now. (ROR, NodeJS, PHP, RN, React)

**Comment by borinmin** [comment]
Hi everyone,  let me briefly introduce myself:

- Borin Min (find more [My Github)](https://borinmin.github.io/)
- Web3 concepts are something quite new to me.
- I have been programming for 6+ years. I can code Python, JS, Java, and some frameworks such Django, Flask, Tensorflow
- Also I have experienced in deep learning  for about 2 years

**Reply by samnang** [reply]
Welcome to Web3 Together. It's interesting to see people who come from different backgrounds.

**Comment by metrey** [comment]
Hello,
- I'm Metrey
- Just see me as a software engineer
- I do Java / PHP / C#, software architecture, product management/lead
- I am some where solving mostly related to Khmer Unicode stuffs, reporting implementation  and blog at https://ask.osify.com
- Web3 is new to me, blockchain is in my interesting topic to go

**Reply by samnang** [reply]
Welcome to Web3 Together. I love to see the same faces that we used to see years ago. I have been off from being involved in the community for a long times. I hope this movement on this topic will bring us back.

**Reply by kruyvanna** [reply]
Idol ❤️

**Comment by meassinal** [comment]
Quick intro about me:
- I'm Sinn (Or Sinal)
- A software developer and product development lead for almost 10 years so far using Ruby on Rails, JavaScript, and Python
- Join the group so I can learn and share new interests in technology especially blockchain

**Reply by samnang** [reply]
Welcome to Web3 Together.  So much joys when I see you guys are interested in this topic. I hope we all will make it fun again.

**Comment by Bunlong** [comment]
Hey

* [Bunlong](https://github.com/bunlong)
* GitHuber and YouTuber
* New in Web3
* ALL

**Reply by samnang** [reply]
Welcome to Web3 Together. Low to see you join us. It has been a while since I have seen some of our faces.

**Comment by daveMcelf** [comment]
Hello, everyone. I saw the post from facebook and want to share my experience here.
- I'm Chanrith.
- Software engineer at a FinTech startup, based in Korea.
- Had some experience with Solidity for a while. But currently mainly focus on Solana ecosystem and SPL from early this year
- I love trying out new tech, so I just get my hand dirty on here and there in everything. 😅

**Reply by samnang** [reply]
Welcome to Web3 Together. I'm glad to hear from some experiences to be a part of the community. I have been playing around a bit on Solidity, then I jumped to token economy in this space, especially DeFi because my background is also building payment systems for FinTech. 

I heard about Solana, but I haven't tried it yet. Recently, it became a network that everybody is talking about it because of Ethereum gas fees.

I hope it would be nice if you could jump in into the discussion when we have a meetup happened.

**Reply by daveMcelf** [reply]
Sure, bong. There are so many diverse and interesting topics that we can learn. I'm also find myself starting to understand the economic of money and finance-related stuff just from getting involve in this space. That's the reason why I love Web3 and the whole DeFi ecosystem. 
I cannot wait for the first meetup and love to hear from others' opinions and the discussion. If time is possible, I will be happy to join.

**Reply by tepkenvannkorn** [reply]
This is also what I am into recently. Let's gather for a meetup!

**Comment by invisal** [comment]
Visal In
- CTO of Groupin
- Zero experience with Web3. Now start to learn it
- I have been programming for 20+ years (10 years professionally). I am into competitive programming. I am familiar with many languages but in the last 4 years, I work mainly with Javascript/Typescript.

**Reply by samnang** [reply]
Welcome to Web3 Together. A lot of things in this new topic, so let's dig in.

**Reply by kruyvanna** [reply]
Idol ❤️

**Comment by sky-king-489** [comment]
Hello seniors,

- I'm Vicheth (or 72)
- Tech Lead at a startup company name [Delicio App](https://delicio.app)
- 0 and newbie to Web3, but love to learn new things.
- I have been working in UX/UI design for 2+ years, programming along for 2+ years (Javascript/Typescript) and going forward to sharp more. I just graduated in 2019 at Faculty of Engineering RUPP, so I'm a freshly newbie.

Thanks bongs bongs 🤝

**Reply by samnang** [reply]
Welcome to Web3 Together. Great to see young people to join the group 👍

**Comment by siveing** [comment]
Hello seniors,

- I'm Siveing
- Offshore developer 
- Had some experience with Scilla for a while, and a little experience with Web3 but I still freshly learn it
- I have been working in MeteorJS for 2+ years and Laravel, Symfony for 1 year

Thanks

**Reply by samnang** [reply]
Welcome to Web3 Together. Great to see you join the group. Enjoy learning!

**Comment by Uysim** [comment]
Hello everyone, Let me introduce myself:
- I'm Uysim (just call me Sim)
- I'm a technical lead of Cloudmattr
- I'm also a technical lead of Growllowing (we are close to lunch)
- I'm new to web3 and happy to explore it
- I'm have been working in software development for about 7 years. Mostly, I work with Ruby on Rails

**Reply by samnang** [reply]
Welcome to Web3 Together. Let's make things great together.

**Comment by Vuthisak** [comment]
Let me start by introducing myself:
  - Vuthisak KHY
  - Android Developer at ABA Bank
  - Web3 is really new to me but want to learn about it.
  - I'm familiar with lots of languages for the last 3 years but now just focusing on Android Development

**Reply by samnang** [reply]
Welcome to Web3 Together. Great to see you join the community.

**Comment by kruyvanna** [comment]
Hi, hello
- Kruy Vanna
- Chief Product Officer at Luma
- Just started to learn about web3 a few months ago
- UI/UX
- Familiar with Java, Swift, Typescript, PHP, Python

**Reply by samnang** [reply]
Welcome to Web3 Together. The community is happy to see you be a part of it.

**Comment by lihourchhin** [comment]
Hi, Hello 
- LyhourChhen 
- Currently, Programmer at @asurraa
- Know a little bit about web3 and blockchain
- Familiar with TypeScripts and Rust.

**Reply by samnang** [reply]
Welcome to Web3 Together. I have never done any Rust codes before. I heard a few strong points to using it compared to other high - level programming languages. Especially some blockchains are programmed in Rust. I hope you can share your Rust experiences in the community. 

We are going to have the first remote meet up on [Saturday 27th, Nov 2021 at 10-11am UTC+7](https://github.com/web3together/web3together/discussions/9). It would be great to see you joining us.

**Comment by saingsab** [comment]
Hi There!

This is Saing and here is my [twitter](https://twitter.com/saingsab)
CTO of [Selendra ](https://github.com/selendra) And Head of Infra at @Koompi 
I've been in touch with Web3 for awhile as well as with Blockchain and Cryptocurrency since 2016.
I have been programming for 6 years now since I am switching from system engineer and infrastructure which is another 4 years. I am enjoy writing Clojure which is my favorite functional programming language. Others language I am touch for supporting the Web3 ecosystem like Solidity for smart contract and Rust programming language. I try to minimal by not learning too many languages so that I can sharpen my skill but still I have to add Nodejs which help me a lot to test the webclient or backend that connect web talk to the data onchain.

**Reply by samnang** [reply]
Welcome to Web3 Together. The community will be at another level when we see more people like you share your experiences of building something with blockchain. Selendra is an interesting project. Your experience with Selendar or blockchain in general, will be very helpful for our community.

I have never touched Clojure before, but I heard a few good things about it because of their functional styles. I used to learn Elixir before, but I have never got a chance to do it as a professional. Great to see more and more people are doing in Rust. 

We are going to have the first remote meet up on [Saturday 27th, Nov 2021 at 10-11am UTC+7](https://github.com/web3together/web3together/discussions/9). It would be great to see you joining us.

**Comment by malinkaphann** [comment]
Hi

I suppose this is where to start.

- Malinka
- A programmer and a noobie in blockchain
- Familiar with C/C++, Java, PHP, Python, Javascript and Go.
- Looking for an opportunity to learn and to contribute by helping to solve problems using this blockchain.

**Reply by samnang** [reply]
Welcome to Web3 Together. Feel free to jump in by starting a new discussion or joining an existing one.

**Comment by kikouch** [comment]
Hi guys!

This is Ki Kouch
- Software Dev in Turnitin, Currently based in Khmer
- Transition from Web Dev (Java, Spring Boot) to Blockchain Dev
- Been self learning about Web3, Blockchain, Solidity, been learning on/off about 6 months
- Working on dApp

**Reply by samnang** [reply]
Welcome to Web3 Together. Feel free to jump in by starting a new discussion or joining an existing one.

**Comment by vatanaksambath** [comment]
Hello!!!

I am Vatanak

- Manager of Application Development at one of the private company based in Phnom Penh
- Interesting in Blockchain and Web3 but just start to learn about it
- I am experienced with lots of languages, but now focusing on C#, JS, Dart, Python, and Matlab
- Looking to gain my knowledge and contribute to Web3 here
